mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/102 main menu look and feel

Updated the main menu to reflect the style of the rest of the ui.

To test:

Start the game and make sure the main menu looks and functions as expected 

Closes #102.
